opium/src/request.mli

```diff
@@ -424,6 +424,17 @@ val content_type : t -> string option
     [Content-Type] set to [content_type]. *)
 val set_content_type : string -> t -> t
 
+(** {3 [authorization]} *)
+
+(** [authorization t] returns the value of the header [Authorization] of the request [t]. *)
+val authorization : t -> Auth.Credential.t option
+
+(** {3 [set_authorization]} *)
```

Comment:
This is only relevant when making requests I think. I'm not very familiar with opium, but is it something that we want to expose or will be used?

Reply:
This is especially useful to build requests for unit tests